# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsPresContext.cpp
# Commit: 5a39eb924188
# Full Hash: 5a39eb924188e504dad274a2148543c8d51dbae8
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 2. Merge FontFaceSetImpl and gfxUserFontSet into a single class. r=emilio
#   
#   This patch merges FontFaceSetImpl and gfxUserFontSet into the same
#   class. This reduces the level of indirection and in theory makes it
#   easier to manage future threading issues.
# ==============================================================================

diff -r 3d30a8d83d77 -r 5a39eb924188 layout/base/nsPresContext.cpp
--- a/layout/base/nsPresContext.cpp	Tue Jun 28 21:46:59 2022 +0000
+++ b/layout/base/nsPresContext.cpp	Tue Jun 28 21:47:00 2022 +0000
@@ -207,7 +207,7 @@
   // If there's a user font set, discard any src:local() faces it may have
   // loaded because their font entries may no longer be valid.
   if (auto* fonts = Document()->GetFonts()) {
-    fonts->GetUserFontSet()->ForgetLocalFaces();
+    fonts->GetImpl()->ForgetLocalFaces();
   }
 
   FlushFontCache();